extensions/amp-story/1.0/amp-story.css

```diff
@@ -187,7 +187,7 @@ amp-sidebar[side][open] {
   max-width: 320px !important;
 }
 
-amp-sidebar[side]{
+amp-sidebar[side] {
```

Comment:
nit: sorry for the churning here; I wonder if we should apply a css class like `.i-amphtml-story-sidebar` to all stories sidebars, and use that for the styles, to lower the specificity ao this may be more easily overridden?

Reply:
This makes sense! Just did the thing.